src/Build/Instance/ProjectInstance.cs

```diff
@@ -1074,8 +1080,14 @@ public bool IsImmutable
         {
             [DebuggerStepThrough]
             get
-            { return InitialTargets; }
-            set { InitialTargets = value; }
+            {
+                return InitialTargets;
+            }
+
+            set
+            {
+                InitialTargets = value;
+            }
```

Comment:
This property looks like a candidate for the single-line version. Would this not fix the warning?

```C#
            get { return InitialTargets; }
            set { InitialTargets = value; }
```

Same for a couple of similar props in this PR.

Reply:
I tried, but I still get an error because of the `[DebuggerStepThrough]` attribute. We need to probably raise another issue for the analyzer.

Reply:
Raised an issue at https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3434